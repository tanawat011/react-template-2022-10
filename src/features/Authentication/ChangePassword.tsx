export const ChangePassword = () => {
  return <>ChangePassword</>
}
