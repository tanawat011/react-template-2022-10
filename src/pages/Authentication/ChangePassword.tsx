import { PATH } from 'Routes'
import { Button } from 'components/Button'

export const ChangePassword = () => {
  return (
    <>
      <div id='change-password-form'>
        <h1>Change Password</h1>

        <form className='flex flex-col'>
          <label id='password'>Old Password</label>
          <input id='password' />
          <label id='newPassword'>New Password</label>
          <input id='newPassword' />

          <div className='flex'>
            <Button link to={`../${PATH.AUTH.LOGIN}`}>
              Submit
            </Button>
          </div>
        </form>
      </div>
    </>
  )
}
