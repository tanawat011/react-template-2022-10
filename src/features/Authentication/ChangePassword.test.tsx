import { render } from '@testing-library/react'

import { ChangePassword } from './ChangePassword'

describe('<ChangePassword />', () => {
  test('match snapshot', () => {
    const { asFragment } = render(<ChangePassword />)
    expect(asFragment()).toMatchSnapshot()
  })
})
