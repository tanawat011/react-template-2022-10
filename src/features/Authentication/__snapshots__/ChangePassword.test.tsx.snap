// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChangePassword /> match snapshot 1`] = `
<DocumentFragment>
  ChangePassword
</DocumentFragment>
`;
